README.md

```diff
@@ -16,180 +18,72 @@ Streisand
+See also:
+* A [more technical list of features](Features.md) 
+* A [more technical list of services](Services.md)
 
-**Note:** Running Streisand against an existing server can be a destructive action! You will be potentially overwriting configuration files and must be certain that you are affecting the correct machine.
+<a id="cloud-providers"></a>
+## Cloud providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Digital Ocean
+* Google Compute Engine (GCE)
+* Linode
+* Rackspace
 
-### Noninteractive Deployment (Advanced) ###
 
-Alternative scripts and configuration file examples are provided for
-noninteractive deployment, in which all of the required information is passed
-on the command line or in a configuration file.
+#### Other providers
+If you are an expert and can set up a *fresh Ubuntu 16.04 server* elsewhere, Streisand includes code for:
 
-Example configuration files are found under `global_vars/noninteractive`. Copy
-and edit the desired parameters, such as providing API tokens and other choices,
-and then run the appropriate script.
+* Using `ssh` to connect to your remote server from your local machine
+* After connecting to your remote server, you can run Streisand *in place*, called "localhost deployment". This converts the server to a Streisand node.
 
-To deploy a new Streisand server:
+"Localhost deployment" often works better than `ssh` deployment. 
 
-      deploy/streisand-new-cloud-server.sh \
-        --provider digitalocean \
-        --site-config global_vars/noninteractive/digitalocean-site.yml
+For more information, see [the advanced installation instructions](Advanced%20installation.md).
 
-To run the Streisand provisioning on the local machine:
+## Installation
 
-      deploy/streisand-local.sh \
-        --site-config global_vars/noninteractive/local-site.yml
+You need command-line access to a Unix system. You can use Linux, BSD, or macOS; on Windows 10, the Windows Subsystem for Linux (WSL) counts as Linux. 
 
-To run the Streisand provisioning against an existing server:
+Once you're ready, see the [full installation instructions](Installation.md).
 
-      deploy/streisand-existing-cloud-server.sh \
-        --ip-address 10.10.10.10 \
-        --ssh-user root \
-        --site-config global_vars/noninteractive/digitalocean-site.yml
 
 Upcoming Features
 -----------------
 * Easier setup.
+* Faster adoption of new censorship-avoidance tools
+
+We're looking for help with both.
 
-If there is something that you think Streisand should do, or if you find a bug in its documentation or execution, please file a report on the [Issue Tracker](https://github.com/StreisandEffect/streisand/issues).
+If there is something that you think Streisand should do, or if you find a bug in its documentation or execution, please file a report on the [Issue Tracker](https://github.com/StreisandEffect/streisand/issues). We appreciate 
```

Comment:
typo: This ends abruptly with just "We appreciate"

Reply:
Fixed.